### PR TITLE
Tarea 4

### PR DESCRIPTION
**Pull request by OlmeiroOrozco**
Task 4: I tried to work with modules js.

### CODE REVIEW

**Comment by emolinaglobant** [comment]
@OlmeiroOrozco , this looks a lot like @juan-hm solution

**Comment by OlmeiroOrozco** [comment]
In fact, I based it on the following repo: https://github.com/accesibleprogramacion/calculadora. 
And I tried to change it with modules and separate things as it should be: in HTML only HTML and the respective changes with Javascript.

I'm sure that we worked under the same example.

File: olmeiro-orozco/tarea-4/results.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  addNumber(number) {
+    // console.log(number);
+    // console.log(this.currentValue);
+    if (number === "." && this.currentValue.includes(".")) return;
+    this.currentValue = this.currentValue.toString() + number.toString();
+    this.showValues();
+  }
+
+  calculateType(type) {
+    // console.log(type);
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-4/results.js]: @OlmeiroOrozco , please remove console logs

File: olmeiro-orozco/tarea-4/calculator.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export class Calculator {
+  add(num1, num2) {
+    return num1 + num2;
+  }
+  minus(num1, num2) {
+    return num1 - num2;
+  }
+  multiply(num1, num2) {
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-4/calculator.js:11]: @OlmeiroOrozco , make sure all class methods are alphabetically sorted

File: olmeiro-orozco/tarea-4/results.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      add: "+",
+      minus: "-",
+      multiply: "*",
+      divide: "/",
+      percent: "%",
+    };
+  }
+
+  addNumber(number) {
+    // console.log(number);
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-4/results.js]: @OlmeiroOrozco , remove console logs

File: olmeiro-orozco/tarea-4/results.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
+import { Calculator } from "./calculator.js";
+
+export class Results {
+  constructor(showOldValue, showCurrentValue) {
+    this.showOldValue = showOldValue;
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-4/results.js]: @OlmeiroOrozco , none of these properties are defined as class members

**Review by emolinaglobant — CHANGES_REQUESTED** [review]

**Review by emolinaglobant — APPROVED** [review]

**Pull request merged**